home/templates/pages/Contactus.html

```diff
@@ -19,23 +17,23 @@ <h1>Contact us</h1>
           <p>For further information, please contact us using the details below:</p>
           <div class="icon-text">
             <i class="fas fa-map-marker-alt"></i>
-            <span> 221 Burwood Hwy, Burwood VIC 3125</span>
+            <span><a href="https://g.co/kgs/abekme1" target="_blank">221 Burwood Hwy, Burwood VIC 3125</a></span>
           </div>
```

Comment:
I don't think a short link to a search for the address on google.co.in is the correct thing to put here, could you try and get a maps.google.com link for the address instead

Reply:
Changed the link as per the requirements